src/nora/components/IconLink/IconLink.js

```diff
@@ -28,34 +28,30 @@ export const IconLink = ({
   iconName,
   iconClassName,
   iconContainerClassName,
-  linkClassName,
-  linkPosition,
+  textClassName,
+  textPosition,
   copy,
   onClick,
 }) => {
-  if (linkPosition === 'left') {
+  if (textPosition === 'left') {
     return (
-      <div className={iconContainerClassName}>
-        <Link href="#" className={linkClassName} onClick={onClick}>
-          {copy}
-        </Link>
+      <Link href="#" className={iconContainerClassName} onClick={onClick}>
+        <div className={textClassName}>{copy}</div>
```

Comment:
I wonder if a `<span>` would be more semantic since these are essentially inline